internal/osbuild2/mkfs_stages_test.go

```diff
@@ -21,25 +21,23 @@ func TestNewMkfsStage(t *testing.T) {
 		UUID:  uuid.New().String(),
 		Label: "test",
 	}
-	btrfsDevices := &MkfsBtrfsStageDevices{Device: *device}
-	mkbtrfs := NewMkfsBtrfsStage(btrfsOptions, btrfsDevices)
+	mkbtrfs := NewMkfsBtrfsStage(btrfsOptions, device)
 	mkbtrfsExpected := &Stage{
 		Type:    "org.osbuild.mkfs.btrfs",
 		Options: btrfsOptions,
-		Devices: btrfsDevices,
+		Devices: Devices{"device": *device},
```

Comment:
I think that it would be good if we had a constructor-like function for `Devices` (e.g. similar to https://github.com/osbuild/osbuild-composer/blob/4a8e5e8b19888face6adb3c7041d55de6d9da7e5/internal/osbuild2/xz_stage_test.go#L28). Personally I'm not a fan of literals with "magic" values such as `device`, because one has to has to know what key to use. Defining the key `device` as a constant would be also an improvement IMO. Especially when this literal with the key is used on 4 places here and in every "filesystem" stage constructor-function.

Reply:
Using more constants makes sense, but let's leave that for future work.